bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
```

Comment:
Consider using a more meaningful name for the `BadCodeExample` class, as it doesn't provide any context about its purpose. [important]